.idea/sonarlint-state.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="SonarLintProjectState">
+    <lastEventPolling>1557393725856</lastEventPolling>
```

Comment:
This timestamp will continuously change for different users, so you will have lots of merge conflicts